contributing-guides/translation-templates/common-arguments.md

```diff
@@ -16,7 +16,7 @@ Only the left-alignment of the header gets lost and has to be re-added again (`|
 | es    | ruta/al/archivo      | ruta/al/directorio     | ruta/al/archivo_o_directorio      | paquete   |                   |
 | fa    |                      |                        |                                   |           |                   |
 | fr    | chemin/vers/fichier  | chemin/vers/dossier    | chemin/vers/fichier_ou_dossier    | paquet    | nom_d_utilisateur |
-| hi    |                      |                        |                                   |           |                   |
+| hi    | फ़ाइल/का/पथ            | निर्देशिका/का/पथ            | फ़ाइल_या_निर्देशिका/का/पथ                 | संकुल      | उपयोगकर्ता-नाम         |
```

Comment:
```suggestion
| hi    | फ़ाइल/का/पथ            | निर्देशिका/का/पथ            | फ़ाइल_या_निर्देशिका/का/पथ                 | पैकेज      | उपयोगकर्ता-नाम         |
```
Would this (पैकेज) be applicable here?

> Note: Don't commit this, Let's hear what others think.

Reply:
I've never heard anyone using the word `पैकेज` in Hindi. I got a Sanskrit word for it: `सम्पुटिततन्त्रांश`, which seems to make the things harder.